packages/clerk-js/src/core/resources/CommercePlan.ts

```diff
@@ -12,14 +12,14 @@ export class __experimental_CommercePlan extends BaseResource implements __exper
   currencySymbol!: string;
   currency!: string;
   description!: string;
-  isActiveForPayer!: boolean;
   isRecurring!: boolean;
   hasBaseFee!: boolean;
   payerType!: string[];
   publiclyVisible!: boolean;
   slug!: string;
   avatarUrl!: string;
   features!: __experimental_CommerceFeature[];
+  subscriptionIdForCurrentSubscriber!: string | undefined;
```

Comment:
I am not sure I am following here. We are asserting that `subscriptionIdForCurrentSubscriber!` is definitely going to be assigned, but then in the constructor we initiate it to `undefined`. 

```suggestion
  subscriptionIdForCurrentSubscriber: string | undefined = undefined;
```

Then we don't need to assign it to undefined in the constructor